src/activities/assignments/AssignmentEntity.js

```diff
@@ -458,6 +465,25 @@ export class AssignmentEntity extends Entity {
 		return this._entity && this._entity.hasActionByName(Actions.assignments.updateCustomAllowableFileType);
 	}
 
+	/**
+	 * @summary Formats action and fields if assignment has been ai inspired
+	 * @param {object} isAiInspired whether this assignment has been ai inspired
+	 */
+	_formatUpdateAiInspiredAction(isAiInspired) {
+		if (typeof isAiInspired === 'undefined') return;
+		if (!this._hasAiInspiredChanged(isAiInspired)) return;
+
+		const action = this._entity.getActionByName(Actions.assignments.updateName);
```

Comment:
I'm confused with this `Actions.assignments.updateName`. Shouldn't this be an ai inspired action?

Reply:
yeah, they all have the same endpoint so I'm re-using this action's url instead of adding another one. Our siren-sdk doesn't actually look at the fields in the action, and hard-codes them anyway.

Reply:
ah ok, thanks for explaining it